app-admin/packagekit*: version bump to 1.1.1

mudler opened the pull request:
Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=587070
* removed nsplugins USE
* 1.1.1 requires at least glib 2.46

Package-Manager: portage-2.2.26

Required for Gnome 3.20

gktrk commented:
@gentoo/proxy-maint @pacho2 @lxnay

gktrk commented:
Is there a particular reason why a single large commit is preferred over one commit per package?

mudler commented:
Nope, just a bad habit when changes are strictly correlated. i can easily split it up if you wish

gktrk commented:
Yes please. Unless these changes need to be atomic (which they don't seem to be), it's better to split them up.

mudler commented:
I could just split the commit into two, 1 for version 1.1.0, and one for 1.1.1 (the 3 packages needs to be bumped together anyways, it's not atomic)

EvaSDK commented on app-admin/packagekit-base/packagekit-base-1.1.1.ebuild:
```diff
@@ -0,0 +1,100 @@
+	prune_libtool_files --all
+
+	dodoc AUTHORS ChangeLog MAINTAINERS NEWS README
+	# Fix QA Notices
```
This is not the way to fix this QA notice. It must be fixed either in an init script if any or in upstream daemon code.

EvaSDK commented:
As already explained by @gktrk, please keep do one commit per atom.
Also do not bump two versions at a time, we are only interested in latest release.

EvaSDK commented:
This looks better. I have 2 questions about your packagekit patch though:
* as it is not Gentoo specific, have you submitted it upstream ? This is very important for us as we do not want to carry them forever, also packagekit upstream is pretty positive about integrating patches. The patch should also be in git format-patch format as it eases the maintenance work.
* does the patch actually keep packagekit working ? It is ok to remove unneeded folder creation in install target but does the daemon actually start/work when such folders are missing ?

mudler commented:
Sorry, was a bit busy lately, daemon seems working normally also after the patch, and pkcon too:

```
pkcon get-details app-text/tree
Risoluzione                   [=========================]         
Recupero dei dettagli         [=========================]         
Interrogazione                [=========================]         
Completato                    [=========================]         
Descrizione del pacchetto
  package:     app-text/tree-1.7.0.amd64
  summary:     
  license:     GPL-2
  group:       unknown
  description: Lists directories recursively, and produces an indented listing of files
  size:        110632 bytes
  url:         http://mama.indstate.edu/users/ice/tree/

```

```
systemctl status packagekit.service                                                              
 packagekit.service - PackageKit Daemon
   Loaded: loaded (/usr/lib64/systemd/system/packagekit.service; static; vendor preset: disabled)
   Active: active (running) since lun 2016-07-04 09:25:42 CEST; 8s ago
 Main PID: 31699 (packagekitd)
   CGroup: /system.slice/packagekit.service
           └─31699 /usr/libexec/packagekitd

lug 04 09:25:42  systemd[1]: Starting PackageKit Daemon...
lug 04 09:25:42  PackageKit[31699]: daemon start
lug 04 09:25:42  systemd[1]: Started PackageKit Daemon.
```

but contents on /var/cache doesn't get created after starting daemon
i'll submit upstream as soon as possible

pacho2 commented:
Did you update the PR changes to achieve that? At least, please add a comment in the packagegit-base. Also remember to provide the Manifest files (maybe that was causing the automatic checks to fail.. but I have tested the details and they are empty :/ now)

As a side note, why don't you use eapi6 for all the ebuilds? And, finally, please RDEPEND on gobject-introspection with := as explained at:
https://wiki.gentoo.org/wiki/Project:GNOME/Gnome_Team_Ebuild_Policies#dev-libs.2Fgobject-introspection:.3D_dependency

Also, probably this would benefit for a := rdepend on networkmanager (its subslot will need to be bumped sooner or later when, for example, we are going to remove the deprecated dbus-glib support on NM)

Thanks

mudler commented:
Sorry, didn't see the answer (Github notifications sucks a little bit). About the PR: i rebased, but Manifest was still there, but the CI showed no output for me too in first place

Adapted to EAPI 6 and added := on nm and gobject :)

pacho2 commented:
The PR check failed... but the errors look not related with this :/

@monsieurp Could you please help me on properly merging this (not like I did the last time ;) You can reach me via mail and I can try :)

Thanks

monsieurp commented:
Could you please rebase against master? I cannot cleanly apply your patch. Thanks.

mudler commented:
@monsieurp didn't see the comment, just rebased :)
